frontend/src/reducers/Meeting.js

```diff
@@ -82,6 +85,48 @@ export default function meeting(state=initialState, action) {
 			return state;
 		case "SEARCH_MEETING":
 			return state;
+
+		*/
+		case "GET_MEETING":
```

Comment:
Same with comment.js, can be removed.
Probably we can use it, if we make some alert of it. However, still need to be optimized